deploy/src/db.ts

```diff
@@ -42,7 +50,8 @@ export class ConfigDB extends KeysDb {
     }
   }
 
-  async disableKey(_id: string, _reason: string): Promise<void> {
-    // do nothing
+  async disableKey(id: string, _reason: string, newStatus: string): Promise<void> {
+    await this.limitsDB.prepare('UPDATE keyStatus SET status = ? WHERE id = ?').bind(newStatus, id).run()
+    await this.limitsDB.prepare('DELETE FROM keyStatus WHERE expiresAt < CURRENT_TIMESTAMP').run()
```

Comment:
on reflect, let's remove this. We can setup another worker to delete these and also `spends` that have expired nightly.